src/inverseRobot.js

```diff
@@ -8,6 +8,16 @@
 
 function inverseRobot(robot) {
   // write code here
+  const inversed = {};
+
+  for (const k in robot) {
+    if (inversed.hasOwnProperty(robot[k])) {
+      return null;
+    }
+    inversed[robot[k]] = k;
```

Comment:
A blank line between a block and the next command
```suggestion
    }

    inversed[robot[k]] = k;
```